v19.2/build-a-python-app-with-cockroachdb-pony.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Build a Python App with CockroachDB and PonyORM
+summary: Learn how to use CockroachDB from a simple Python application with PonyORM.
+toc: true
+twitter: false
+---
+
+<div class="filters filters-big clearfix">
+    <a href="build-a-python-app-with-cockroachdb.html"><button style="width: 20%" class="filter-button">Use <strong>psycopg2</strong></button></a>
+    <a href="build-a-python-app-with-cockroachdb-sqlalchemy.html"><button style="width: 20%" class="filter-button">Use <strong>SQLAlchemy</strong></button></a>
```

Comment:
nit: The `Use SQLAlchemy` tab looks horizontally off-center in the online preview.